krabs/krabs/etw.hpp

```diff
@@ -337,10 +335,9 @@ namespace krabs { namespace details {
     }
 
     template <typename T>
-    void trace_manager<T>::stop_trace()
+    void trace_manager<T>::unregister_trace()
```

Comment:
looks like `trace_.registrationHandle_` is no longer being checked or assigned after this change. is that deliberate?

Reply:
So... it wasn't at the time....  But it is now.

The registrationHandle is only used by enable_providers which is only ever called immediately after register_trace.  (So it could probably be refactored out entirely).

query_stats actually superfluously passes the registrationHandle to ControlTrace() but it also passes the trace name. And this functions ignores the handle in this case.

I'm going to update query_stats so that it also can be called before open/start. 
Currently I plan to swallow the 'trace not found' case - and return a 0-filled stats structure. 
You'll be able to check stats.BufferCount to determine if a trace is running.

This will also handle the previous edge case when krabs would throw an unknown exception if query_stats is called after ControlTrace(STOP)  e.g. logman stop MyTrace -ets